linux_os/guide/services/ntp/chronyd_no_chronyc_network/bash/shared.sh

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_rhel
```

Comment:
I think the remediation script can be also applicable to Fedora.

Reply:
Added Fedora and OL in ed89b5901de6c7147a29ade4d931ff716a7733b1. Also rebase in master.